### DISCUSSION TITLE
New Sum helper

### DISCUSSION THREAD

**Discussion by gjohansson-ST** [home-assistant/architecture#822] [General] [answered]
## Context

Suggestions from the community to get a simple helper that sums the state of other entities
https://community.home-assistant.io/t/wth-helpers-can-do-calculus-but-not-arithmetic/467631

## Proposal

Create a new helper `sum` that consumes the state of other entities, sums it and presents it as new state for new sensor.

Would be pretty similar to what `min_max` helper is doing.

Not too uncommon that ones wants to sum up the value of sensors to use them to present various things in frontend or to easily use it as a trigger in an automation.

PR already make in draft: https://github.com/home-assistant/core/pull/79398

**Comment by frenck** [comment, marked as answer]
I don't think an architectural discussion is needed for this helper, as it not specifically working around other architectural design decisions (like the attribute helper discussion you created as well).
